<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeletePointsLayout</class>
 <widget class="QWidget" name="DeletePointsLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>257</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter ID of the point you want to edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pointToEditLineEdit">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Enter new coordinates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xEditLineEdit">
        <property name="text">
         <string>X Coordinate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zEditLineEdit">
        <property name="text">
         <string>Z Coordinate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
